code/modules/antagonists/dwarves/dwarves.dm

```diff
@@ -0,0 +1,29 @@
+/datum/team/dwarves
+	name = "Dwarven Empire"
+	show_roundend_report = TRUE
+
+/datum/antagonist/dwarf
+	name = "Ash Walker"
+	job_rank = ROLE_LAVALAND
+	show_in_antagpanel = FALSE
+	prevent_roundtype_conversion = FALSE
+	antagpanel_category = "Dwarves"
+	var/datum/team/dwarves/dwarf_team
+
+/datum/antagonist/dwarf/create_team(datum/team/team)
+	if(team)
+		dwarf_team = team
+		objectives |= dwarf_team.objectives
+	else
+		dwarf_team = new
+
+/datum/antagonist/dwarf/get_team()
+	return dwarf_team
+
+/datum/antagonist/dwarf/greet()
+	to_chat(owner.current, "<span class='warning'>As a dwarf you must follow these 5 tenats:</span><br>")
+	to_chat(owner.current, "<span class='warning'>You may never kill or maim another dwarf intentionally outside of a duel!</span><br>")
+	to_chat(owner.current, "<span class='warning'>Fortune is worth more than spilt blood!</span><br>")
+	to_chat(owner.current, "<span class='warning'>Expand your emprire and crush anyone who resists!</span><br>")
+	to_chat(owner.current, "<span class='warning'>Trade and barter with foreigners, force shall be applied only when they provoke you!</span><br>")
+	to_chat(owner.current, "<span class='warning'>You may never leave this ashen land for it is your homeland!</span><br>")
```

Comment:
we should not do this anymore (repeated to_chat), also this has a greet and a mob spawn text so both of these are insanely long and will play remove one or the other

Reply:
it works in game